torch/_inductor/compile_fx.py

```diff
@@ -1130,9 +1130,17 @@ def codegen_and_compile(
                         colored=True,
                     ),
                 )
-                inductor_post_grad_graph_str = gm.print_readable(
-                    print_output=False, include_stride=True, include_device=True
-                )
+
```

Comment:
is it possible to push this logic inside print_readable? and we can pass fast_sympy_print=True?
```
gm.print_readable(print_output=False, include_stride=True, include_device=True, fast_sympy_print=True)
```
The reason is that i expect i will have to apply this on more places than just this .
namely all structured tracing logs as well.